<launch>

	<node pkg="vectornav" type="vn200_node" name="vectornav"  output="screen" >
	  <remap from="imu/data"        to="imu/data" />
	  <remap from="imu/magnetic"    to="imu/magnetic" />
	  <remap from="imu/temperature" to="imu/temperature" />
	  <remap from="gps/fix"         to="gps/fix" />
	  <remap from="gps/time"        to="gps/time" />	  
	  
	  <param name="serial_port" type="string" value="/dev/ttyUSB0" />
	  <param name="serial_baud" type="int"    value="115200" />
	  <param name="poll_rate"   type="int"    value="40" />
	  
	  <param name="imu/frame_id" type="string" value="imu" />
	  <param name="gps/frame_id" type="string" value="gps" />
	  
	  <param name="time_reference_source" type="string" value="UTC" />
	  <param name="time_offset"           type="double" value="0.0" />
	</node>	
	
</launch>


